python_cdk/src/detectors/python-cdk-emr-auth-ec-2-key-pair-or-kerberos/python-cdk-emr-auth-ec-2-key-pair-or-kerberos_compliant.py

```diff
@@ -0,0 +1,26 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-cdk-emr-auth-ec-2-key-pair-or-kerberos@v1.0 defects=0}
+import aws_cdk as cdk
+from aws_cdk import Stack
+from aws_cdk.aws_emr import CfnCluster
+from aws_cdk.aws_emr.CfnCluster import InstanceGroupConfigProperty,KerberosAttributesProperty
+from aws_cdk import aws_emr as emr
+	 
+class CdkStarterStack(cdk.Stack):
+    def __init__(self, scope: cdk.App, id: str):
+      super(scope, id)   
+        
+    # Compliant: `kerberos_attributes` is set.
```

Comment:
"The CfnCluster instantiation sets the kerberos_attributes property with KerberosAttributesProperty, including kdc_admin_password and realm."